plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/ArtifactRetrieverIntegrationTest.java

```diff
@@ -56,9 +56,20 @@ public void testGetGuava19() throws InvalidVersionSpecificationException {
     Assert.assertEquals(19, guava.getMajorVersion());
     Assert.assertEquals(0, guava.getMinorVersion());
   }
-  
+
   @Test
   public void testGetLatestArtifactVersion() {
+    ArtifactVersion version = ArtifactRetriever.DEFAULT.getLatestVersion(
+        "com.google.cloud", "google-cloud-dns");
+    Assert.assertNotNull(version);
+    
+    // version 0.26 or later
+    double x = Double.parseDouble(version.getMajorVersion() + "." + version.getMinorVersion());
```

Comment:
Just pointing out that this won't work if they release '0.1xx' later (extremely unlikely though).

Reply:
Ah, I had looked at this earlier thinking _I know there must be an issue with this_.  Would be safer to compare the components separately as integers.

Reply:
Generally, this double comparison doesn't work. Another case is like when comparing 0.26 and 0.9. (0.9 is an earlier version.)